src/main/res/values/themes.xml

```diff
@@ -15,8 +15,6 @@
             <item name="colorControlHighlight">@color/simple_alpha33</item>
         <item name="alertDialogTheme">@style/AppCompatAlertDialogStyle</item>
         <item name="actionOverflowButtonStyle">@style/WhiteActionButton</item>
-        <item name="android:textColorPrimary">@color/white</item>
-        <item name="popupTheme">@style/ThemeOverlay.AppCompat.Light</item>
```

Comment:
Put this (popupTheme) in when fixing overflow menu styling earlier but it turns out with other changes I made it is no longer needed.